src/app/application.cpp

```diff
@@ -144,13 +144,15 @@ Application::Application(const QString &id, int &argc, char **argv)
     QPixmapCache::setCacheLimit(PIXMAP_CACHE_SIZE);
 #endif
 
+    m_defaultProfileFolderFound = m_commandLineArgs.profileDir.isEmpty() && QDir(QCoreApplication::applicationDirPath()).exists(DEFAULT_PORTABLE_MODE_PROFILE_DIR);
```

Comment:
>Also "relative fastresume" option has no meaning without "profile" option

Sorry, it seems I'm wrong. It can have a meaning even with default profile (not portable).

So now your code seems to be correct, but `m_defaultProfileFolderFound` now has other meaning (default portable profile folder is found and isn't overriden by command-line parameters, so "portable mode enabled").

Reply:
Thanks for the suggestions. I changed the code, please let me know if I should change anything else.